src/routes/docs/products/functions/develop/+page.markdoc

```diff
@@ -683,6 +683,7 @@ These are provided alongside any custom headers sent to the function.
 |---------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `x-appwrite-trigger`        |  Describes how the function execution was invoked. Possible values are `http`, `schedule` or `event`.                                                      |
 | `x-appwrite-event`          |  If the function execution was triggered by an event, describes the triggering event.                                                                |
+| `x-appwrite-key`            |  The dynamic API key is used for server authentication. You don't need to create it. Configure function permission scopes to access your project services, like users and buckets. |
```

Comment:
> You don't need to create it. 

Let's get rid of this. Every one of these headers are automatically generated and injected by Appwrite.

> Configure function permission scopes to access your project services, like users and buckets.

Instead of this, is there anywhere else where we give detailed information about what a dynamic api key is?